apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -42,7 +42,8 @@ const TutorialExplorer = React.createClass({
     backButton: React.PropTypes.bool,
     roboticsButtonUrl: React.PropTypes.string,
     showSortBy: React.PropTypes.bool.isRequired,
-    disabledTutorials: React.PropTypes.arrayOf(React.PropTypes.string).isRequired
+    disabledTutorials: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
+    sortByPopularity: React.PropTypes.bool
```

Comment:
Will we want to be even more flexible in the future? Should this just be `sortBy: React.PropTypes.string.oneOf(['popularityRank', 'default'])`?